helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -747,7 +752,7 @@ public void reset() {
     reset(true);
   }
 
-  void reset(boolean isShutdown) {
+  public void reset(boolean isShutdown) {
```

Comment:
is this necessary?

Reply:
yes, there are others who are calling this method.

Reply:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project helix-core: Compilation failure: Compilation failure: 
[ERROR] /Users/kdesai/experiment/helix/helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java:[384,16] reset(boolean) has private access in org.apache.helix.manager.zk.CallbackHandler